docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponents.tsx

```diff
@@ -23,6 +23,7 @@ export function CustomizationComponents() {
           );
         },
         ErrorView: ({ children }) => {
+          // children contains a default error component based on the LivenessError, you can use our default rendered components or render your own error message
```

Comment:
Prefer to add this line in the docs above the example to prevent drift between the example code and the docs themselves

Reply:
Will do